calico-cloud/threat/container-threat-detection.mdx

```diff
@@ -90,6 +90,24 @@ If a malicious or suspicious program is run within the cluster, it will be repor
 
 ![malware-alert-example](/img/calico-cloud/malware-alert-example.png)
 
+
+### Create Exceptions for runtime logs
+
+If there is a program running within the cluster that is performing a suspicious action but you want to except it from create a runtime log, and thus from triggering a Security Event. Then you can create an Excetpion in the Runtime Security CR: 
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: operator.tigera.io/v1
+kind: RuntimeSecurity
+metadata:
+  name: default
+spec:
+  runtimeExceptionList:
```

Comment:
Surely we should provide an actual example here?

Reply:
this was a mistake. I must've been on the wrong branch working on https://github.com/tigera/docs/pull/1692 and added this by mistake. Reverting this